tock/tock/static/sass/_entries.scss

```diff
@@ -1,12 +1,12 @@
 .entries {
 	.entry:last-child {
-		border-bottom: 2px solid $color-gray-light;
```

Comment:
These fixes didn't really have anything to do with the homepage redesign, but it took me all of fifteen minutes to remove these old variables (and hardcoded hex codes!) in favor of USWDS 2.0 variables.